config/ini_test.go

```diff
@@ -67,7 +69,7 @@ func TestIni(t *testing.T) {
 		t.Error(pi)
 		t.Fatal(err)
 	}
-	if iniconf.String("runmode") != "dev" {
+	if iniconf.String("runmode") != beego.DEV {
```

Comment:
I think here we still van keep use the string "dev". As `iniconf.String("")` is testing the string result. it doesn't make sense use beego.DEV